### PR TITLE
Karma tests debugging

### PR DESCRIPTION
**Pull request by merapi**
## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [ ] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

**Comment by swissspidy** [comment]
Just converting to draft to not unnecessarily create Percy snapshots and the like

File: packages/karma-puppeteer-launcher/src/index.cjs [hunk 1]
```diff
@@ -50,6 +50,7 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
         // See http://crbug.com/715363
         // We use this flag to work-around this issue.
         '--disable-dev-shm-usage',
+        '--force-gpu-mem-discardable-limit-mb',
```
Comment (swissspidy) [inline packages/karma-puppeteer-launcher/src/index.cjs]: This also seems to help in Brittany's PR, #10095

Comment (merapi) [reply packages/karma-puppeteer-launcher/src/index.cjs]: I wanted to check what makes the difference, flags or disabled tests/other fixes.
Looks like this flag alone doesn't do much: https://github.com/google/web-stories-wp/runs/4774876792?check_suite_focus=true
I'll try the second.

Comment (merapi) [reply packages/karma-puppeteer-launcher/src/index.cjs]: Locally I removed both flags from #10095 and it's still ✅ 
image
I will just check it on CI and then look into disabled tests/other fixes.

Comment (swissspidy) [reply packages/karma-puppeteer-launcher/src/index.cjs]: Running tests locally hasn't been an issue AFAIK, only running on CI

Comment (merapi) [reply packages/karma-puppeteer-launcher/src/index.cjs]: I've had a few buffer/memory-related messages locally, like this one from the latest CI run (with those 2 flags):

image

So far it looks like the flags are not an instant fix, but they might lower the overall amount of used memory/buffer size produced.

That `268435456` is https://source.chromium.org/chromium/chromium/src/+/main:content/browser/devtools/devtools_http_handler.cc;l=83

**Pull request closed without merging**